src/views/Home/components/ColoredWordHeading.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Colors, Heading, TextProps } from '@pancakeswap/uikit'
+import useTheme from 'hooks/useTheme'
+
+interface HeadingProps extends TextProps {
+  text: string
+  firstColor?: keyof Colors
+}
+
+const ColoredWordHeading: React.FC<HeadingProps> = ({ text, firstColor, mb = '24px', ...props }) => {
+  const { theme } = useTheme()
+  const split = text.split(' ')
+  const firstWord = split[0]
```

Comment:
```
const [firstWord] = text.split(' ')
```

Reply:
We need `split` just below, probably need to keep the variable here.

Reply:
I wonder how it would work with Japanese/Chinese where there are no spaces 😅 
Did a quick test - seems to be coloring whole sentence and `remainingWords` becomes `''`.
But I guess there's no need to cover for that, not sure we'll even have translations by the time (although if we'd keep the page for long time maybe worth checking what to do with it)

Reply:
oh lol, just realized it is upgrade of old PurpleWordHeading, so it was like this for a long time 😄

Reply:
Yes indeed, I just renamed/changed it a bit to make it more generic.